Makefile

```diff
@@ -139,8 +139,8 @@ else ifeq ($(AUTO_TEST), boot)
 endif
 
 .PHONY: gdb_server
-gdb_server: build
-	@cd kernel && cargo osdk run $(CARGO_OSDK_ARGS) -G --vsc --gdb-server-addr :1234
+gdb_server: initramfs $(CARGO_OSDK)
+	@cargo osdk run $(CARGO_OSDK_ARGS) -G --vsc --gdb-server-addr :1234
 
```

Comment:
Since `1234` are used in two places, I think defining a variable for it may be better.
```Makefile
GDB_TCP_PORT=1234
```